### PR TITLE
Pyflyte register optionally activates schedule

### PR DESCRIPTION
**Pull request by kumare3**
# TL;DR
register now supports a new option
```bash
pyflyte register module.py --activate-launchplans
```

It is also ofcourse possible to deactivate a launchplan
```bash
pyflyte launchplan every_min_installer --deactivate
```

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [x] Code documentation added
 - [x] Any pending items have an associated Issue

## Tracking Issue
https://github.com/flyteorg/flyte/issues/3109

### CODE REVIEW

File: flytekit/clis/sdk_in_container/register.py [hunk 1]
```diff
@@ -113,6 +108,14 @@
     is_flag=True,
     help="Execute registration in dry-run mode. Skips actual registration to remote",
 )
+@click.option(
+    "--activate-launchplans",
+    "--activate-launchplan",
+    default=False,
+    is_flag=True,
+    help="Flip current active version to the newly registered version of an active LaunchPlan. This combines to "
```
Comment (eapolinario) [inline flytekit/clis/sdk_in_container/register.py:116]: typo: to -> two

Comment (eapolinario) [reply flytekit/clis/sdk_in_container/register.py:116]: Consider rephrasing this, something like: "Activate newly registered Launchplans. This operation deactivates previous versions of Launchplans."

**Review by wild-endeavor — DISMISSED** [review]
+1 but title of pr is not correct right?  it won't automatically do it, you need to add

File: flytekit/clis/sdk_in_container/register.py (hunk 1 above)
Comment (kumare3) [reply flytekit/clis/sdk_in_container/register.py:116]: Done

**Review by eapolinario — APPROVED** [review]

**Review by pingsutw — APPROVED** [review]

**Pull request merged**